patches/nodebb-plugin-composer-default+9.2.4.patch

```diff
@@ -74,67 +75,144 @@ index e9a8ca6..ab81fb5 100644
 +		.slider.round:before {
 +			border-radius: 50%;
 +		}
- 	}
++	}
++
++  .anon-switch {
++    display: flex;
++  }
++
++  /* The switch - the box around the slider */
++  // https://www.w3schools.com/howto/howto_css_switch.asp
++  .switch {
++    position: relative;
++    display: inline-block;
++    width: 60px;
++    height: 34px;
++  }
++
++  /* Hide default HTML checkbox */
++  .switch input {
++    opacity: 0;
++    width: 0;
++    height: 0;
++  }
++
++  /* The slider */
++  .slider {
++    position: absolute;
++    cursor: pointer;
++    top: 0;
++    left: 0;
++    right: 0;
++    bottom: 0;
++    background-color: #ccc;
++    -webkit-transition: .4s;
++    transition: .4s;
++  }
++
++  .slider:before {
++    position: absolute;
++    content: "";
++    height: 26px;
++    width: 26px;
++    left: 4px;
++    bottom: 4px;
++    background-color: white;
++    -webkit-transition: .4s;
++    transition: .4s;
++  }
++
++  input:checked + .slider {
++    background-color: #2196F3;
++  }
++
++  input:focus + .slider {
++    box-shadow: 0 0 1px #2196F3;
++  }
++
++  input:checked + .slider:before {
++    -webkit-transform: translateX(26px);
++    -ms-transform: translateX(26px);
++    transform: translateX(26px);
++  }
  
  	.display-scheduler {
+         display: flex;
 diff --git a/node_modules/nodebb-plugin-composer-default/static/lib/composer.js b/node_modules/nodebb-plugin-composer-default/static/lib/composer.js
-index 46334e7..315f065 100644
+index 46334e7..caa39b9 100644
 --- a/node_modules/nodebb-plugin-composer-default/static/lib/composer.js
 +++ b/node_modules/nodebb-plugin-composer-default/static/lib/composer.js
-@@ -206,6 +206,7 @@ define('composer', [
+@@ -206,6 +206,8 @@ define('composer', [
  			tags: data.tags || [],
  			modified: !!((data.title && data.title.length) || (data.body && data.body.length)),
  			isMain: true,
 +			isPrivate: data.isPrivate,
++      isAnon: data.isAnon,
  		};
  
  		({ pushData } = await hooks.fire('filter:composer.topic.push', {
-@@ -431,6 +432,7 @@ define('composer', [
+@@ -343,7 +345,9 @@ define('composer', [
+ 		submitBtn.on('click', function (e) {
+ 			e.preventDefault();
+ 			e.stopPropagation();	// Other click events bring composer back to active state which is undesired on submit
+-
++      if ($('.isAnon').is(':checked')) {
```

Comment:
This seems like logs for debugging, maybe remove for production

Reply:
Removed it from the package.